app/routes/__init__.py

```diff
@@ -1,4 +1,10 @@
-__all__ = [
-    "static",
-    "post"
-]
+import os
+
+__all__ = []
+
+
+# This seems like a hack but it doesn't work with the default __all__ and this way this file doesn't
+# Need to be updated for each new route module.
+for i in os.listdir('./app/routes'):
+    if i != '__init__.py' and i.endswith('.py'):
+        __all__.append(i[0:-3])
```

Comment:
Why not just `from app.routes import *`?

Reply:
I tried and it didn't work.

Reply:
The thing is, the import isn't being used for the defined functions, the import has side effects (in the form of `@app.route`), and every route *must* be imported for it to work.

Reply:
This sounds like a job for [blueprints](http://flask.pocoo.org/docs/0.12/blueprints), possibly. I'm still opposed to just blindly loading every Python file in a directory into the module's namespace. Adding the file to `__all__` manually would be safer and cleaner, though a bit more inconvenient.

Reply:
I'll look at blueprints in a bit then.